pages/resource/[id].tsx

```diff
@@ -0,0 +1,68 @@
+import type {NextPage} from 'next'
+import {gql, useQuery} from "@apollo/client";
+import React, {useEffect, useState} from "react";
+import { useRouter } from 'next/router'
+import Popup from "../../components/popup";
+import base45 from "base45";
+import QRCode from "react-qr-code";
+
+const Resource: NextPage = () => {
+    const router = useRouter()
+    const { id } = router.query
+    const [result, setResult] = useState('');
+
+    const fetchQr = async () => {
+    const data = {
+      "data": {
+        "reflow_data_to_post": {
+          "id": id,
+          "recurseLimit": 10,
+          "unwind": true
+        },
+        "reflow_endpoint": "https://reflow-demo.dyne.org/api/json/trace",
+        "sawroomEndpoint": "http://195.201.41.35:8008"
```

Comment:
This should be configurable but with default set to the default sawroom instance